src/inputs/input-textarea.component.ts

```diff
@@ -21,4 +21,18 @@ export class InputTextareaComponent extends InputBase implements OnInit {
     }
 
     public config: InlineTextareaConfig;
+
+    public onKeyPress(event) {
+        if (this.config.saveOnEnter) {
+            if (event.charCode === 13 && !event.shiftKey) {
+                this.save();
+                this.onEscape(event);
+            }
+        } else {
+            if (event.charCode === 13 && event.shiftKey) {
+                this.save();
+                this.onEscape(event);
+            }
```

Comment:
if and else do the same action (this.save() and this.onEscape(event)) then is not necessary if-else. You would rethink the condition using an OR operator maybe.

If you have problems with this logic, contact me and can rethink together.